src/frontend/apps/desk/src/features/teams/api/useTeams.tsx

```diff
@@ -2,18 +2,44 @@ import { UseQueryOptions, useQuery } from '@tanstack/react-query';
 
 import { APIList, fetchAPI } from '@/api';
 
-interface TeamResponse {
+interface Access {
+  id: string;
+  role: string;
```

Comment:
should be an Enum

Reply:
https://github.com/numerique-gouv/people/pull/31/commits/31c9e6cfa3e116c7ff64a68c31e4339de408c3d6